system_tests/monitoring.py

```diff
@@ -172,7 +177,8 @@ def test_create_and_delete_metric_descriptor(self):
         )
 
         descriptor.create()
-        descriptor.delete()
+        retry_404(descriptor.delete)()
```

Comment:
You're retrying on 404 for deletion of a metric descriptor, but #2092 is reporting a 503 for deletion of a group.